ui/src/layout/Menu.js

```diff
@@ -44,9 +54,13 @@ const Menu = ({ onMenuClick, dense, logout }) => {
     <MenuItemLink
       key={resource.name}
       to={`/${resource.name}`}
+      activeClassName={classes.active}
       primaryText={translatedResourceName(resource, translate)}
       leftIcon={
-        (resource.icon && createElement(resource.icon)) || <ViewListIcon />
+        path.search(resource.name) > -1
```

Comment:
This could lead to false positives. Maybe a better way is to use a regex making sure the path matches something like: `/#/${resource.name}/`

Reply:
Okay i will use the regex